azurerm/resources.go

```diff
@@ -101,6 +124,34 @@ var (
 		//Container Registry Resources
 		ContainerRegistry:        containerRegistries,
 		ContainerRegistryWebhook: containerRegistryWebhooks,
+		//Storage Resources
+		StorageAccount:   storageAccounts,
+		StorageQueue:     storageQueues,
+		StorageFileShare: storageFileShares,
+		StorageTable:     storageTables,
+		//Database Resources- mariadb
+		MariadbConfiguration:      mariadbConfigurations,
+		MariadbDatabase:           mariadbDatabases,
+		MariadbFirewallRule:       mariadbFirewallRules,
+		MariadbServer:             mariadbServers,
+		MariadbVirtualNetworkRule: mariadbVirtualNetworkRules,
+		//Database Resources - mysql
+		MysqlConfiguration:      mysqlConfigurations,
+		MysqlDatabase:           mysqlDatabases,
+		MysqlFirewallRule:       mysqlFirewallRules,
+		MysqlServer:             mysqlServers,
+		MysqlVirtualNetworkRule: mysqlVirtualNetworkRules,
+		//Database Resources - postgresql
+		PostgresqlConfiguration:      postgresqlConfigurations,
+		PostgresqlDatabase:           postgresqlDatabases,
+		PostgresqlFirewallRule:       postgresqlFirewallRules,
+		PostgresqlServer:             postgresqlServers,
+		PostgresqlVirtualNetworkRule: postgresqlVirtualNetworkRules,
+		//Database Resources- sql
```

Comment:
Same for the comments. He missing space.

And also fix the ones that are there already on this function for what I see eheh.